react/__tests__/OrderItems.test.tsx

```diff
@@ -95,4 +97,83 @@ describe('OrderItems', () => {
 
     console.error = oldConsoleError
   })
+
+  it('should update itemList when updateItemWithId is called', async () => {
+    const oldConsoleError = console.error
+    console.error = () => {}
```

Comment:
Why is that?

Reply:
Ah, nice catch. I had put this because the testing library was complaining that a state update was happening outside an `act` call and I wasn't being able to get rid of it because the lib didn't support `async` actions at that time. Now that we updated the lib version it worked fine.

It is present on purpose in the first test though because it expects the component to throw, and the lib ends up printing the whole error on the screen if I don't silence `console.error`.